command/server.go

```diff
@@ -511,8 +511,15 @@ func (c *ServerCommand) Run(args []string) int {
 		barrierSeal = vault.NewAutoSeal(vaultseal.NewTestSeal(nil))
 	} else {
 		// Handle the case where no seal is provided
-		if len(config.Seals) == 0 {
+		switch len(config.Seals) {
+		case 0:
 			config.Seals = append(config.Seals, &server.Seal{Type: vaultseal.Shamir})
+		case 1:
```

Comment:
What if they had multiple seal stanzas that were disabled? Should this logic instead be like, if they have no seal stanzas, or if they do but they're all disabled? 

Kind of an edge case, but could happen. Like maybe if they'd originally used a GCP key, then AWS, and now back to Shamir.

If that seems too edge-case-y, this seems totally fine.

Reply:
See https://github.com/hashicorp/vault/blob/master/command/server/config.go#L802 in conjunction with https://github.com/hashicorp/vault/blob/master/command/server/config.go#L834

Reply:
Gotcha, thanks!